guides/common/modules/proc_applying-errata-to-a-host-sles15.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Applying_Errata_to_a_Host_Running_SLES_15_{context}"]
+= Applying Errata to a Host Running SLES 15
+
+Use these procedures to review and apply errata to a host running SUSE Linux Enterprise Server 15.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Content Hosts* and select the host you want to apply errata to.
+. Navigate to the *Errata* tab to see the list of errata.
```

Comment:
```suggestion
. Navigate to the *Errata* tab to view the list of errata.
```
I think view is better than see, but no massively strong opinions. In the GUI examples in IBM style.. it is always "view". A non-blocker all the same.

Reply:
Applied.